libcomposefs/lcfs-writer.c

```diff
@@ -471,9 +471,16 @@ struct lcfs_node_s *lcfs_node_new(void)
 		errno = ENOMEM;
 		return NULL;
 	}
-
+	
+	memset( node, 0, sizeof(struct lcfs_node_s));
```

Comment:
No need to memset, calloc() already does that.

Reply:
Thank you. These are not required anymore.